lib/jblond/Diff/Renderer/Text/UnifiedCli.php

```diff
@@ -49,7 +49,7 @@ public function render(): string
         if (!isset($this->options['cliColor'])) {
             return $this->output();
         }
-        if (isset($this->options['cliColor']) && $this->options['cliColor'] == 'simple') {
+        if (isset($this->options['cliColor']) && $this->options['cliColor'] === true) {
```

Comment:
I think `if ($this->options['cliColor']) {` is sufficient.

The variable should be set already by MainRendererAbstract and acting on any truthy value is ok.

Reply:
I changed that